sdk/program/src/epoch_rewards.rs

```diff
@@ -38,6 +39,10 @@ pub struct EpochRewards {
     /// Whether the rewards period (including calculation and distribution) is
     /// active
     pub active: bool,
+
+    /// Excess padding, should be set to `0`s
+    #[serde(skip)]
+    pub _padding: ZeroedPadding<15>,
```

Comment:
nit: These fields should be private.

Reply:
That would require adding a constructor to this type. It's already a breaking change so we could add that.